iteration.py

```diff
@@ -1,6 +1,17 @@
+""" module docstring """
 import math
 
 
-def sin_(x: float) -> float:
+def sin_(number):
+    """ function docstring """
+    sine = 0
+    i = 0
+    while True:
+        max_sine = sine
+        sine += ((-1) ** (i)) * ((number ** (2 * i + 1))
```

Comment:
Finding the correct value of Taylor series is also not as easy as it seems. You can significantly optimize this code using previous power and factorial values.